.circleci/docker/common/install_rocm.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+set -ex
+
+install_ubuntu() {
+    apt-get update
+    apt-get install -y wget
+    apt-get install -y libopenblas-dev
+
+    # Need the libc++1 and libc++abi1 libraries to allow torch._C to load at runtime
+    apt-get install libc++1
+    apt-get install libc++abi1
+
+    DEB_ROCM_REPO=http://repo.radeon.com/rocm/apt/debian
+    # Add rocm repository
+    wget -qO - $DEB_ROCM_REPO/rocm.gpg.key | apt-key add -
+    echo "deb [arch=amd64] $DEB_ROCM_REPO xenial main" > /etc/apt/sources.list.d/rocm.list
+    apt-get update --allow-insecure-repositories
+
+    DEBIAN_FRONTEND=noninteractive apt-get install -y --allow-unauthenticated \
+                   rocm-dev \
+                   rocm-utils \
+                   rocfft \
+                   miopen-hip \
+                   rocblas \
+                   hipsparse \
+                   rocrand \
+                   hipcub \
+                   rocthrust \
+                   rccl \
+                   rocprofiler-dev \
+                   roctracer-dev
```

Comment:
We don't have to fix this in this diff, but similar to the other installation scripts, you should clear the apt cache when you're done to reduce the size of the docker image.